app/models/package_manager/npm.rb

```diff
@@ -89,10 +89,14 @@ def self.licenses(latest_version)
     end
 
     def self.versions(project)
-      project["versions"].map do |k, _v|
+      # npm license fields are supposed to be SPDX expressions now https://docs.npmjs.com/files/package.json#license
```

Comment:
Should this be accepting a `name` argument like all the others?

Reply:
yes good catch.